modules/openapi-generator/src/main/resources/go/go.mod.mustache

```diff
@@ -3,7 +3,9 @@ module {{gitHost}}/{{gitUserId}}/{{gitRepoId}}{{#isGoSubmodule}}/{{packageName}}
 go 1.13
 
 require (
+    {{#hasOAuthMethods}}
```

Comment:
minor suggestion: use tab instead of 4-space to make it align properly with line 8

Reply:
Updated the indentation, thank you so much for spotting that